pyhomematic/devicetypes/sensors.py

```diff
@@ -190,6 +190,27 @@ class MotionV2(Motion, HelperSabotage):
     """Motion detection version 2."""
     pass
 
+class MotionIP(HMBinarySensor, HMSensor, HelperLowBat, HelperSabotage):
+    """Motion detection indoor (rf ip)"""
+
+    def __init__(self, device_description, proxy, resolveparamsets=False):
+        super().__init__(device_description, proxy, resolveparamsets)
+
+        # init metadata
+        self.BINARYNODE.update({"MOTION_DETECTION_ACTIVE": [1]})
+        self.BINARYNODE.update({"MOTION": [1]})
```

Comment:
Line 200 and 201 can be condensed to one. So one dictionary containing both keys.